src/config.ts

```diff
@@ -28,6 +18,7 @@ export const config = {
         required: true,
         token: process.env.TOKEN || 'kd-token',
         secret: process.env.SECRET_KEY || 'pandora@drive', // TODO: Don't use static value in production! remove from source control!
+        daysExpires: +(process.env.TOKEN_DAYS_EXPIRES || 30),
         saml: {
             entryPoint: process.env.SAML_ENTRY_POINT || 'http://localhost:8080/simplesaml/saml2/idp/SSOService.php',
             issuer: process.env.SAML_ISSUER || 'http://localhost:3000/metadata.xml',
```

Comment:
issuer should also be changed to `http://localhost:8080/auth/metadata.xml`, missed it too...